website/react-magma-docs/src/pages/data-visualization/chart-types.mdx

```diff
@@ -0,0 +1,304 @@
+      <h2>Donut</h2>
+      <p>
+        A donut chart is a type of pie chart with a hole in the middle. It shows
+        the relationship of parts to a whole and the percentage each value
+        contributes to the total.
+      </p>
+      <a href="../chart-demos/#donut">View Demo</a>
+    </div>
+  </div>
+  <div class="item">
+    <figure>
+      <img
+        src="../../images/chart-types/stacked-bar.jpg"
+        alt="GATSBY_EMPTY_ALT"
+      />
+    </figure>
+    <div class="description">
+      <h2>Stacked Bar</h2>
+      <p>
+        A stacked bar chart, also called a stacked bar graph or stacked column
+        chart, compares multiple variables over time.
+      </p>
+      <a href="../chart-demos/#bar_(stacked)">View Demo</a>
+    </div>
+  </div>
+  <div class="item">
+    <figure>
+      <img src="../../images/chart-types/bullet.jpg" alt="GATSBY_EMPTY_ALT" />
+    </figure>
+    <div class="description">
+      <h2>Bullet</h2>
+      <p>
+        Bullet charts are commonly used in dashboards to effectively compare
+        metrics against target benchmarks or predefined ranges.
+      </p>
+      <a href="../chart-demos/#bullet">View Demo</a>
+    </div>
+  </div>
+  <div class="item">
+    <figure>
+      <img
+        src="../../images/chart-types/stacked-area.jpg"
+        alt="GATSBY_EMPTY_ALT"
+      />
+    </figure>
+    <div class="description">
+      <h2>Stacked area</h2>
+      <p>
+        A stacked area chart is a graph that shows how the values of multiple
+        groups change over time.
+      </p>
+      <a href="../chart-demos/#area_(stacked)">View Demo</a>
+    </div>
+  </div>
+  <div class="item">
+    <figure>
+      <img src="../../images/chart-types/meter.jpg" alt="GATSBY_EMPTY_ALT" />
+    </figure>
+    <div class="description">
+      <h2>Meter</h2>
+      <p>
+        Meter charts are used to measure the rate of change of a quantity
+        against pre-defined targets.
+      </p>
+      <a href="../chart-demos/#meter">View Demo</a>
+    </div>
+  </div>
+  <div class="item">
+    <figure>
+      <img src="../../images/chart-types/gauge.jpg" alt="GATSBY_EMPTY_ALT" />
+    </figure>
+    <div class="description">
+      <h2>Gauge</h2>
+      <p>
+        A gauge chart, also known as a dial or speedometer chart, displays a
+        single data value quantitatively.
+      </p>
+      <a href="../chart-demos/#gauge">View Demo</a>
+    </div>
+  </div>
+</div>
+
+## Correlations
+
+Correlation charts show the relationship between two or more variables.
+
+<div class="container">
+  <div class="item">
+    <figure>
+      <img src="../../images/chart-types/scatter.jpg" alt="GATSBY_EMPTY_ALT" />
+    </figure>
+    <div class="description">
+      <h2>Scatter</h2>
+      <p>
+        Scatter plots use data points to plot two measures along a scale,
+        allowing exploration of correlations between different variables.
+      </p>
+      <a href="../chart-demos/#scatter">View Demo</a>
```

Comment:
These links don't seem to scroll to the correct part of the page 🤔

Reply:
New issue was created to address this later: https://github.com/cengage/react-magma/issues/1549